static/app/utils/replays/hydrateBreadcrumbs.tsx

```diff
@@ -27,8 +27,12 @@ export default function hydrateBreadcrumbs(
         }
         return {
           ...frame,
+          // Logcat and Timber are considered a console frame instead of a custom breadcrumb frame
```

Comment:
```suggestion
          // Logcat and Timber are used for mobile replays and are considered a console frame instead of a custom breadcrumb frame
```